gamemodes/husklesph/entities/entities/ph_disguise.lua

```diff
@@ -1,48 +1,40 @@
-
 AddCSLuaFile()
 
-DEFINE_BASECLASS( "base_anim" )
+DEFINE_BASECLASS("base_anim")
 
-ENT.PrintName		= "Disguise Ent"
-ENT.Author			= "Mechanical Mind"
-ENT.Information		= ""
-ENT.Category		= ""
+ENT.PrintName = "Disguise Ent"
+ENT.Author = "Mechanical Mind"
+ENT.Information = ""
+ENT.Category = ""
 
-ENT.Editable			= true
-ENT.Spawnable			= true
-ENT.AdminOnly			= false
-ENT.RenderGroup 		= RENDERGROUP_BOTH
+ENT.Editable = true
+ENT.Spawnable = true
+ENT.AdminOnly = false
+ENT.RenderGroup = RENDERGROUP_BOTH
 
 function ENT:SetupDataTables()
-
 end
 
-
 function ENT:Initialize()
-
 	if SERVER then
-		self:SetMoveType( MOVETYPE_NONE )
-		self:SetSolid( SOLID_NONE )
+		self:SetMoveType(MOVETYPE_NONE)
+		self:SetSolid(SOLID_NONE)
 	end
-
 end
 
-if ( CLIENT ) then
-
+if (CLIENT) then
```

Comment:
Any chance you can try and nix the unnecessary parens too?

Reply:
Definitely, just need to find them all.